festim/boundary_conditions/fluxes/surface_kinetics.py

```diff
@@ -0,0 +1,136 @@
+from festim import FluxBC
+from fenics import *
+import sympy as sp
+
+
+class SurfaceKinetics(FluxBC):
+    """
+    FluxBC subclass allowing to include surface processes on in 1D H transport simulations.
+
+    d c_s / dt = k_bs l_abs c_m (1 - c_s/N_s) - k_sb c_s (1 - c_b/N_b) + J_vs
+
+    -D * grad(c) * n = -l_abs * d c / dt - k_bs l_abs c_m (1 - c_s/N_s) + k_sb c_s (1 - c_b/N_b)
```

Comment:
Sure, but I doubt on `l_abs`, which is not strictly defined in papers. For example, @ehodille [used](https://iopscience.iop.org/article/10.1088/1741-4326/ad2a29) different values for `l_abs` in these equations. In the second eq.: `l_abs` - the distance between two IS in W, whereas in the first one: `l_abs=N_s/N_b` - distance between subsurface IS and adsorption sites. As I remember, in some other works this characteristic distance was the same in both eqs. I assume that one value for `l_abs` can be used if `N_s=N_b^(2/3)`, `l_abs=N_b^(-1/3)`.

Reply:
We need to clarify this and cite the paper from which this exact model is taken from. 

Another discrepancy with the paper from Etienne you cite is that the coverage term in `\varphi_{gas}` is squared and not in the current implementation

Reply:
i see `(1-c_b/N_b)` in your equation, what's `c_b`?

Reply:
It's a typo. It should be `c` - solute concentration (I used earlier `c_b` where `b` standed for the bulk).

Reply:
Can we just define the rate constant for the bulk-to-surface transition in `m/s`? That way, the ODE for the surface concentration would become: $\frac{dc_s}{dt}=k_{bs}c_m(1-\frac{c_s}{N_s})-k_{sb}c_s + J_{vs}$. For example:
```python
def k_bs(T, cs):
    nu=10e13
    lambda1 = 110e-12
    E_bs = 0.2
    return nu*lambda1*f.exp(-E_bs/F.k_B/T)
``` 

Thus, the users would have to provide `l_abs` (can be differ from `lambda1` for the H transport BC.

Reply:
I don't see why this would be an issue

Reply:
So, I used a similar notation to Etienne's model, but kept the $(1-c_m/n_{IS})$ term. As long as $c_m << n_{IS}$, the models are equal. Otherwise, I couldn't reproduce the results from [this](https://www.sciencedirect.com/science/article/pii/S2352179120300272) paper on H absorption in Ti.

Reply:
Thanks! I'll close this comment to open a new one.